### PR TITLE
Client & Courses Section

### PR DESCRIPTION
**Pull request by amanvision**

### CODE REVIEW

File: app/Course.php [hunk 1]
```diff
@@ -3,8 +3,13 @@
 namespace App;
 
 use Illuminate\Database\Eloquent\Model;
+use Spatie\Activitylog\Traits\LogsActivity;
 
 class Course extends Model
 {
```
Comment (amanvision) [inline app/Course.php:9]: Please use protected $guarded instead $fillable,

File: app/Http/Controllers/Admin/AdminController.php [hunk 2]
```diff
@@ -178,4 +178,24 @@ private function syncPermissions(Request $request, $user)
...
+            // $request->flash();
+            return response()->json([
+                'success'=>false,
+                'eroror' => $validator->errors('profile')
+            ]);
+        }
+
+        return response()->json([
+            'success' =>true,
+            $request->profile->extension()
```
Comment (amanvision) [inline app/Http/Controllers/Admin/AdminController.php:197]: What is the purpose of returning $request->profile->extendsion()

File: app/Http/Controllers/Admin/CourseController.php [hunk 3]
```diff
@@ -57,7 +96,9 @@ public function show($id)
      */
     public function edit($id)
     {
-        //
+        $universities = DB::table('Universities')->select('id','name')->get();
```
Comment (amanvision) [inline app/Http/Controllers/Admin/CourseController.php:99]: Please use model syntax

File: app/Http/Controllers/Admin/UniversityController.php [hunk 4]
```diff
@@ -69,7 +105,31 @@ public function edit($id)
      */
     public function update(Request $request, $id)
     {
-        //
+        $validator = Validator::make($request->all(),['profile' => 'required|mimes:jpeg,png,jpg,gif,svg']);
```
Comment (amanvision) [inline app/Http/Controllers/Admin/UniversityController.php:108]: Please add size validation like Image size must be less than 5 MB.

File: app/University.php [hunk 5]
```diff
@@ -3,8 +3,13 @@
...
-
+use Spatie\Activitylog\Traits\LogsActivity;
 class University extends Model
 {
-    //
+    protected $table = "universities";
+    use LogsActivity;
+    protected static $logOnlyDirty = true;
+    protected static $logFillable = true;
+    protected $fillable = ['name'];
```
Comment (amanvision) [inline app/University.php:13]: Use $guarded instead $fillable,

File: routes/web.php [hunk 6]
```diff
@@ -126,7 +126,9 @@
         ]
     ]);
 });
+//profile pic
 
+Route::post('/profile','Admin\AdminController@chageProfilePic')->name('profilePic');
```
Comment (amanvision) [inline routes/web.php:131]: Move this line under in above code module.

File: app/Http/Controllers/Admin/AdminController.php (hunk 2 above)
Comment (pardeep4561) [reply app/Http/Controllers/Admin/AdminController.php]: I working on it is used for testing purpose

**Pull request merged**